lib/datasource/gitlab-packages/readme.md

```diff
@@ -0,0 +1,40 @@
+[GitLab Packages API](https://docs.gitlab.com/ee/api/packages.html) supports looking up package versions from [all types of packages registry supported by GitLab](https://docs.gitlab.com/ee/user/packages/package_registry/index.html) and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+
+To specify which specific repository should be queried when looking up a package, the `depName` should be formed with the project path first,
+then the `:` and finally the package name.
+
+As an example, `gitlab-org/ci-cd/package-stage/feature-testing/new-packages-list:@gitlab-org/nk-js` would look for the`@gitlab-org/nk-js` packages in the generic packages repository of the `gitlab-org/ci-cd/package-stage/feature-testing/new-packages-list` project.
+
+If you are using a self-hosted GitLab instance, please note the following requirements:
+
+- If you are on the `Free` edition, this datasource requires at least GitLab 13.3
+- If you are on the `Premium` or the `Ultimate` edition, this datasource requires at least GitLab 11.8, but GitLab 12.9 or more is recommended if you have a lot of packages with different names on the same project
```

Comment:
```suggestion
- If you are on the `Premium` or the `Ultimate` edition, this datasource requires at least GitLab 11.8, but GitLab 12.9 or more is recommended if you have a lot of packages with different names in the same project
```

Would `in the same project` be more correct than `on the same project`? Not fully sure about this suggestion though...

Reply:
Mmh, referring to https://english.stackexchange.com/questions/5881/on-a-project-vs-in-a-project I think in this case ´in´ is better.